src/test-utils/render.tsx

```diff
@@ -13,8 +14,6 @@ import { HashRouter } from 'react-router-dom'
 import store from 'state'
 import ThemeProvider from 'theme'
 
-import catalog from './locales/en-US'
```

Comment:
why did this change?

Reply:
It was just moved to an absolute path for consistentcy.